src/components/Clipboard/Clipboard.tsx

```diff
@@ -37,6 +37,11 @@ export function Clipboard(props: ClipboardProps) {
 
     if (!displayedContent || !rawContent) {
       setError(true);
+
+      setTimeout(() => {
+        setError(false);
+      }, 3000);
+
```

Comment:
Wow! 🤔 I am not sure to follow that.

Reply:
I can remove, i think it was done so the message does not stay for too long